extensions/control-plane/provision/gcs-provision/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright (c) 2022 Google LLC
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Google LCC - Initial implementation
+ *
+ */
+
+plugins {
+    `java-library`
+}
+val failsafeVersion: String by project
+val googleCloudLibariesVersion: String by project
+
+dependencies {
+    api(project(":spi:common:core-spi"))
+    implementation(project(":core:common:util"))
+    implementation(project(":extensions:common:configuration:filesystem-configuration"))
+
+    implementation(project(":extensions:common:gcp:gcp-lib"))
+
+    api("dev.failsafe:failsafe:${failsafeVersion}")
+    implementation(platform("com.google.cloud:libraries-bom:${googleCloudLibariesVersion}"))
```

Comment:
is it really necessary to pull in the entire BOM, which also pulls in Guava and many other things? We prefer having only explicit dependencies. EDC being a platform, we need to be extremely wary of dependency hell.